tests/happy-path.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { test, expect, _electron as electron } from '@playwright/test';
+import { mkdtemp, rm } from 'node:fs/promises';
+import { tmpdir } from 'node:os';
+import { sep, join } from 'node:path';
+
+let installDir: string;
+
+
+test.beforeAll(async () => {
+  try {
+    installDir = await mkdtemp(`${tmpdir()}${sep}drupal-temp-`);
+    console.log(`Temporary directory created at: ${installDir}`);
```

Comment:
We can probably remove the `console.log()` calls, and we can also remove the try-catch structures because, if these fail, then we probably don't want to run the test at all.

So these methods can just be as simple as:

```
installDir = await mkdtemp(...);
```